Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.m

```diff
@@ -215,15 +215,17 @@ - (void)onOutputReceived:(NSString *)output {
         if (__self.currentOutputId == previousOutputId && (__self.simulatorState >= AppLaunched && __self.simulatorState != Completed)) {
             NSString *testClass = (__self.currentClassName ?: __self.previousClassName);
             NSString *testName = (__self.currentTestName ?: __self.previousTestName);
+            BOOL testStartedTruely = [self didTestsStart];
```

Comment:
Can you use a clear name for this variable? Something like `testsReallyStarted`? It's a minor nit, but it reads a bit weird. Thanks!